api/src/lib/auth.ts

```diff
@@ -65,7 +66,7 @@ export const getCurrentUser = async (
       })
 
       if (!user) {
-        logger.error(`User not found for passageId: ${passageId}`, event)
+        logger.error({ custom: event }, 'User not found.')
         return null
       }
 
```

Comment:
Might also be worth adding a log message with the error in the `catch` block on line 95? Just on the off chance that the entire `try` is erroring?

Reply:
Ah! I missed this catch block. No wonder the `AuthenticationError` was not being logged in datadog. Makes a ton more sense now.

Reply:
Addressed in a separate PR: https://github.com/usdigitalresponse/cpf-reporter/pull/265